docs/tutorials/moving-pictures/moving-pictures.md

```diff
@@ -0,0 +1,336 @@
+
+You can install Qurro using [pip](https://pip.pypa.io/en/stable/):
+
+```
+pip install numpy
+pip install qurro
+```
+
+A python version of at least 3.5 is required to use Qurro. (You can check this
+by running `conda info | grep python`; if you're in the latest QIIME 2 conda
+environment, you should already be using a good enough version of Python.)
+
+Once you've installed Qurro, let's get QIIME 2 to recognize it. Run the
+following command in a terminal:
+
+```
+qiime dev refresh-cache
+```
+
+To verify that Qurro was installed, you can run the following command:
+
+```
+qiime qurro --help
+```
+
+If this command succeeds, you should see information about Qurro's QIIME 2
+plugin.
+
+## Input files (what we'll need to run Qurro)
+
+If you've completed the DEICODE tutorial already, you can follow along here
+(using the output files you produced from DEICODE).
+If you haven't completed the DEICODE tutorial already, you can download the
+necessary files for this tutorial here:
+
+- `table.qza` [view](https://view.qiime2.org/?src=https%3A%2F%2Fdocs.qiime2.org%2F2019.4%2Fdata%2Ftutorials%2Fmoving-pictures%2Ftable.qza) | [download](https://docs.qiime2.org/2019.4/data/tutorials/moving-pictures/table.qza)
+- `sample-metadata.tsv` [download](https://data.qiime2.org/2019.4/tutorials/moving-pictures/sample_metadata.tsv)
+- `taxonomy.qza` [view](https://view.qiime2.org/?src=https%3A%2F%2Fdocs.qiime2.org%2F2019.4%2Fdata%2Ftutorials%2Fmoving-pictures%2Ftaxonomy.qza) | [download](https://docs.qiime2.org/2019.4/data/tutorials/moving-pictures/taxonomy.qza)
+- `ordination.qza` [view](https://view.qiime2.org/?src=http%3A%2F%2Fbiocore.github.io%2Fqurro%2Ftutorials%2Fmoving-pictures%2Fdata%2Fordination.qza) | [download](http://biocore.github.io/qurro/tutorials/moving-pictures/data/ordination.qza)
+- `biplot.qzv` [view](https://view.qiime2.org/?src=http%3A%2F%2Fbiocore.github.io%2Fqurro%2Ftutorials%2Fmoving-pictures%2Fdata%2Fbiplot.qzv) | [download](http://biocore.github.io/qurro/tutorials/moving-pictures/data/biplot.qzv)
+
+## Running Qurro
+
+Since we'll be working with DEICODE output (i.e. feature loadings in a
+compositional biplot), we'll need to use the
+`qiime qurro unsupervised-rank-plot` command. (The `supervised-rank-plot`
+command is for working with feature differentials.)
+
+```
+qiime qurro unsupervised-rank-plot \
+    --i-table table.qza \
+    --i-ranks ordination.qza \
+    --m-sample-metadata-file sample-metadata.tsv \
+    --m-feature-metadata-file taxonomy.qza \
+    --o-visualization qurro-plot.qzv
+```
+
+##### Output Artifacts
+
+- `qurro-plot.qzv` [view](https://view.qiime2.org/?src=http%3A%2F%2Fbiocore.github.io%2Fqurro%2Ftutorials%2Fmoving-pictures%2Fdata%2Fqurro-plot.qzv) | [download](http://biocore.github.io/qurro/tutorials/moving-pictures/data/qurro-plot.qzv)
+
+You just generated your first Qurro plot! `qurro-plot.qzv` is a `.qzv` file --
+that is, a QIIME 2 visualization. You can view it either by running `qiime tools
```

Comment:
No comma after `is`

Reply:
replaced `that is, ` with `in other words, `. I think this might be a bit more reasonable.